### DISCUSSION TITLE
What is the best way of inserting an A3 page in the document?

### DISCUSSION THREAD

**Discussion by lleky** [joaomlourenco/novathesis#363] [Q&A]
I have some schematics I want to include in the appendix of my thesis, and in order to be able to see clearly everything it has to be on an A3 page. I'm having trouble inserting an A3 page in the document without messing the title format and page number. I want the page to be horizontal while the number and the title of the Appendix to remain at the top right side and the page number to be on the bottom centered. I have tried various methods I found online, but without any success. I would appreciate if somebody could help me with this issue. Thank you.

**Comment by joaomlourenco** [comment]
Can you tell me/us what you have tried?  Using `pdfpages` I assume?
Which was your best attempt? Please provide the details on what you did to replicate your best result? And then make it clear what is still missing?!
